pkg/handlers/helper.go

```diff
@@ -139,48 +137,46 @@ func alertName(a template.Alert) (*string, error) {
 	return nil, fmt.Errorf("no alertname defined in alert")
 }
 
-// SendServiceLog sends a servicelog notification for the given alert
 func (o *ocmsdkclient) SendServiceLog(summary, firingDesc, resolveDesc, clusterID string, severity v1alpha1.NotificationSeverity, logType string, references []v1alpha1.NotificationReferenceType, firing bool) error {
```

Comment:
Here too (`clusterID` becomes `clusterUUID`)

Reply:
good catch, actually, I was just trying to build the same `LogEntry` pattern as in the original  `SendServiceLog()` function. 
```
sl := ocm.ServiceLog{
		ServiceName:   consts.ServiceLogServiceName,
		ClusterUUID:   clusterID,
		Summary:       summary,
		InternalOnly:  false,
		Severity:      severity,
		LogType:       logType,
		DocReferences: references,
}
```
But yeah, we should rename the variable to maintain consistency.